web/src/aws-exports.ts

```diff
@@ -34,7 +34,9 @@ const config = {
   //zincOxideIngestion: import.meta.env.VITE_ZINC_OXIDE_INGESTION,
   zincQuotaThreshold: import.meta.env.VITE_ZINC_QUOTA_THRESHOLD,
   enableAnalytics: import.meta.env.VITE_ZINC_ANALYTICS_ENABLED,
-  isZincObserveCloud: import.meta.env.VITE_ZINCOBSERVE_CLOUD,
+  isZincObserveCloud: import.meta.env.VITE_ZINCOBSERVE_CLOUD
+    ? import.meta.env.VITE_ZINCOBSERVE_CLOUD
+    : "false",
```

Comment:
```suggestion
  isZincObserveCloud: String(import.meta.env.VITE_ZINCOBSERVE_CLOUD && import.meta.env.VITE_ZINCOBSERVE_CLOUD);
```
Does it have to be `string`?